service/src/main/java/uk/gov/hmcts/reform/fpl/service/additionalapplications/ConfirmApplicationReviewedService.java

```diff
@@ -0,0 +1,78 @@
+package uk.gov.hmcts.reform.fpl.service.additionalapplications;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.common.AdditionalApplicationsBundle;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.fpl.model.event.ConfirmApplicationReviewedEventData;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static org.apache.commons.lang3.ObjectUtils.isEmpty;
+import static uk.gov.hmcts.reform.fpl.enums.YesNo.NO;
+import static uk.gov.hmcts.reform.fpl.enums.YesNo.YES;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.asDynamicList;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.element;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.findElement;
+
+@Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class ConfirmApplicationReviewedService {
+    public static final String ADDITIONAL_APPLICATION_REVIEWED = "ADDITIONAL_APPLICATION_REVIEWED";
+
+    public Map<String, Object> initEventField(CaseData caseData) {
+        Map<String, Object> resultMap = new HashMap<>();
+
+        final List<Element<AdditionalApplicationsBundle>> applicationsBundlesToBeReviewed =
+            getApplicationsToBeReviewed(caseData);
+
+        if (isEmpty(applicationsBundlesToBeReviewed)) {
+            resultMap.put("hasApplicationToBeReviewed", NO);
+        } else {
+            resultMap.put("confirmApplicationReviewedList",
+                asDynamicList(applicationsBundlesToBeReviewed, AdditionalApplicationsBundle::toLabel));
+            resultMap.put("hasApplicationToBeReviewed", YES);
+        }
+
+        return resultMap;
+    }
+
+    private List<Element<AdditionalApplicationsBundle>> getApplicationsToBeReviewed(CaseData caseData) {
+        return caseData.getAdditionalApplicationsBundle().stream()
+            .filter(bundleElement -> !YES.equals(bundleElement.getValue().getApplicationReviewed()))
```

Comment:
`NO.equals(bundleElement.getValue().getApplicationReviewed())` ?

Reply:
I think it need to be `!YES.equals` here. The value `applicationReviewed` is not set when the additional application bundle is just uploaded, while it will be set to `YES` if a judge reviewed it.